x-pack/plugin/profiler/src/main/java/org/elasticsearch/xpack/profiler/TransportGetProfilingAction.java

```diff
@@ -126,6 +126,9 @@ private void searchEventGroupByStackTrace(
     ) {
         long start = System.nanoTime();
         GetProfilingResponseBuilder responseBuilder = new GetProfilingResponseBuilder();
+        int exp = eventsIndex.getExponent();
+        responseBuilder.setIsSampled(exp != 1);
```

Comment:
Nit: We could instead encapsulate this in a method in `EventsIndex`:

```java
    public boolean isSampled() {
        return exponent != 1;
    }
```

Reply:
Correction: This is not the case, and I have fixed the code now.